pkg/debug/scripts/collect-info.sh

```diff
@@ -212,6 +212,8 @@ collect_network_info()
 collect_pillar_backtraces()
 {
     echo "- pillar backtraces"
+    # send http-debug port open command and do other things while it is opening
+    eve http-debug > /dev/null 2>&1
```

Comment:
Please in a separate function, like collect_pillar_memory_dump() or something.

Reply:
Done